agent/ansible/pbench/agent/roles/epel_repo_install/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: install the EPEL repo
+  package:
+    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
+    state: latest
+  when: ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
+  
```

Comment:
I'm not fluent in Ansible Galaxy, and probably it's more trouble than it'd be worth to package a separate "collection"? But since this role (and maybe others) seem to be identical for Agent and Server sides, would it be possible/practical to publish a single role that can be referenced by both sides? (And potentially used elsewhere?)

Reply:
Track in #2293.